### PR TITLE
add/data_grid

### PR DESCRIPTION
**Pull request by Tkachenko01001**
https://app.clickup.com/t/8692wp6a7

### CODE REVIEW

File: src/pages/crm-orders-page/constants.ts
Comment (pqoqubbw) [inline src/pages/crm-orders-page/constants.ts:1]: shouldn't these statuses be taken from generated types from the backend?

File: src/pages/crm-orders-page/dataGridTable.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      field: 'id',
+      headerName: 'Номер замовлення',
+      width: 160,
+      sortable: false,
+      renderCell: (params) => (
+        <Box>
+          <Typography fontWeight={600} color="accent.darker">
+            {`#${params.value}`}
+          </Typography>
+        </Box>
```
Comment (pqoqubbw) [inline src/pages/crm-orders-page/dataGridTable.tsx]: why do you need `Box` here?

File: src/pages/crm-orders-page/dataGridTable.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    },
+    {
+      field: 'price_order',
+      headerName: 'Загальна сума',
+      width: 150,
+      sortable: false,
+      renderCell: (params) => (
+        <Box>
+          <Typography>{formattedTotalPrice(params.value)}</Typography>
+        </Box>
```
Comment (pqoqubbw) [inline src/pages/crm-orders-page/dataGridTable.tsx]: same here

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/pages/crm-edit-order-page/utils.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+export const orderStatusArray = Object.entries(ORDER_STATUS).map(
+  ([key, value]) => ({
+    status: key,
+    text: value.text,
+    style: value.style
+  })
+)
+
+/* eslint-disable complexity */
```
Comment (Tkachenko01001) [inline src/pages/crm-edit-order-page/utils.ts:13]: I haven't figured out how to simplify this code yet

File: src/pages/crm-orders-page/constants.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+type OrderStatusValue = {
+  text: string
+  style: {
+    backgroundColor: string
+    color: string
+  }
+}
+
+export const ORDER_STATUS: { [key in OrdersStatus]: OrderStatusValue } = {
+  'new': {
```
Comment (pqoqubbw) [inline src/pages/crm-orders-page/constants.ts]: this value should be enum not string [FROM_BACKEND.STATUS]: ...

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

**Review by pqoqubbw — APPROVED** [review]

**Pull request merged**